helm/reana/README.md

```diff
@@ -129,3 +129,4 @@ This Helm automatically prefixes all names using the release name to avoid colli
 | `workspaces.paths`                                       | List of additional workspace paths as strings. Each mount string is composed by a key `hostPath`(path to the directory to be mounted from the Kubernetes nodes) and a cluster_pod_mountpath (path inside the cluster containers where the `mountPath` will be mounted) e.g. `hostPath:mountPath`. The first value listed will be the default workspace root path. Any POSIX filesystem mounted on cluster nodes is supported | None |
 | `interactive_sessions.cronjob_schedule`                  | Cron format string describing how often interactive session cleanup should be performed. | "0 3 * * *"                                 |
 | `interactive_sessions.maximum_inactivity_period`         | Set a limit in days for the maximum inactivity period of interactive sessions. After this period interactive sessions will be automatically closed. To disable autoclosure and allow interactive sessions to run forever, use value "forever". | forever |
+| `ui_launcher_repositories`                               | Array of demo examples to show in the launch page in the UI. Each demo repository is composed of `name`, `url`, and `image_url` for an example image of the repository.                                                                                                     | [] |
```

Comment:
This needs to be updated to show the new value name and to also include `specification`